src/Stratis.Bitcoin.IntegrationTests/RPC/RPCTestsMutable.cs

```diff
@@ -21,11 +21,18 @@ public void TestRpcGetBalanceIsSuccessful()
                 builder.StartAll();
                 RPCClient rpcClient = node.CreateRPCClient();
 
-                TestHelper.MineBlocks(node, 2);
-                TestHelper.WaitLoop(() => node.FullNode.GetBlockStoreTip().Height == 2);
+                int maturity = (int)KnownNetworks.RegTest.Consensus.CoinbaseMaturity;
```

Comment:
Can you please use `new BitcoinRegTest().Consensus.CoinbaseMaturity;`, we're trying to move away from these static networks.

```suggestion
                int maturity = (int)new BitcoinRegTest().Consensus.CoinbaseMaturity;
```

Reply:
I think its ok `KnownNetworks` are created specifically for tests (I dont think its static)